src/core/CoolBoardActuator.cpp

```diff
@@ -60,105 +60,101 @@ bool CoolBoardActuator::doAction(JsonObject &root, uint8_t hour,
       // temporal actor
       if (this->secondaryType == "hour") {
         // hour actor
-        if (root[this->primaryType].success()) {
-          // mixed hour actor
-          this->mixedHourAction(hour, root[this->primaryType].as<float>());
-        } else {
+        // if (root[this->primaryType].success()) {
```

Comment:
why is this commented out ?

Reply:
Because for now actuators don't works with sample range

Reply:
remove the code then